.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -7,6 +7,10 @@ assignees: ''
 
 ---
 
+<!---
```

Comment:
Not sure how to change the ordering, but we can also point to the repo here in the template: https://github.com/babel/babel/blob/master/.github/ISSUE_TEMPLATE/config.yml as an external link, depending on what they see

![image](https://user-images.githubusercontent.com/588473/82681275-8d7c3900-9c1b-11ea-9944-ea68ca65dfad.png)

Reply:
Cool, did not know this was a thing!

Reply:
Good idea :+1: Maybe we could put the message in a gist and link it?